roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -919,11 +919,13 @@ def _create(self, fname):
         '''call oc create on a filename'''
         return self.openshift_cmd(['create', '-f', fname])
 
-    def _delete(self, resource, rname, selector=None):
+    def _delete(self, resource, rname=None, selector=None):
         '''call oc delete on a resource'''
-        cmd = ['delete', resource, rname]
+        cmd = ['delete', resource]
         if selector:
             cmd.append('--selector=%s' % selector)
+        elif rname:
```

Comment:
Yes. if you try to specify both a name and a selector `oc` commands will give you an error. The order I chose was based on the code I saw in `_get()`.